### ISSUE TITLE
Dynamically Scaling Kneall Stronghold

### ISSUE THREAD

**Issue by Wingysam** [GalaxyRoblox/Issues-Public#24]
Finish testing and release the updated Kneall Stronghold that scales its difficulty to the amount of DPS in the server.

**Comment by PataNautic** [comment]
Should Stronghold servers be locked after the stronghold spawns to prevent distortion of the scaled DPS? If people that already joined that stronghold server disconnect/quit, they should be allowed to rejoin the server; perhaps a server browser for PvE events, but this is beyond the scope of this item.

**Comment by Wingysam** [comment]
The difficulty scales at the beginning of every wave, so it's not really a big concern. Yes you could all despawn right before every wave but then you don't get as much reward.

**Comment by Wingysam** [comment]
![Image](https://github.com/GalaxyRoblox/Issues-Public/assets/18403742/ce341327-e39e-412e-8e80-07ba677b03b6)

**Comment by PataNautic** [comment]
I suggested this at some point but I don't know if it was in Dev Discord or not
I would like to try and introduce additional methods to obtain a reward that aren't purely damaging the base (or damaging it at all), but I don't know how to properly go about that

**Comment by Jaxumi92** [comment]
I know someone earlier was talking about it, but what about bringing a more special class of ships only for SH (IE CANNOT BE USED IN  MAIN). These ships would automatically over time generate a "participation point" that gets converted into percentage. These ships would be designed for fleet support and buffing alone only in SH. Essentially, lets just take three completely random and brand new ship ideas here. 
Nomuafi would be a Fleet Tank designation with turrets specially made to draw aggro from whatever it is shooting, forcing the AI to engage that ship. It would be INSANELY tanky, but extremely slow. 
Zapuno would be a Repair Ship, designed with zero offensive capabilities. It's job is to keep the full fleet repaired, having repair turrets that first focus hull, then shields.
Asfiago would be the real anti-ship support. Designed to obliterate the larger NPCs, it would do that job extremely well, but have absolute glass for its armor, and fire about as frequently as a potato pc trying to run the most graphics intensive game you can think of.

Just an idea, but could work with what you were thinking pata.

**Comment by Wingysam** [comment]
@Jaxumi92 I think you didn't mean to close the issue?

**Comment by Jaxumi92** [comment]
Nope. Github is weird man, I meant to reply to Pata. Idk where the closure thing came from

**Comment by PataNautic** [comment]
> I know someone earlier was talking about it, but what about bringing a more special class of ships only for SH (IE CANNOT BE USED IN MAIN). These ships would automatically over time generate a "participation point" that gets converted into percentage. These ships would be designed for fleet support and buffing alone only in SH. Essentially, lets just take three completely random and brand new ship ideas here. Nomuafi would be a Fleet Tank designation with turrets specially made to draw aggro from whatever it is shooting, forcing the AI to engage that ship. It would be INSANELY tanky, but extremely slow. Zapuno would be a Repair Ship, designed with zero offensive capabilities. It's job is to keep the full fleet repaired, having repair turrets that first focus hull, then shields. Asfiago would be the real anti-ship support. Designed to obliterate the larger NPCs, it would do that job extremely well, but have absolute glass for its armor, and fire about as frequently as a potato pc trying to run the most graphics intensive game you can think of.
> 
> Just an idea, but could work with what you were thinking pata.

It would be an interesting place to test more team based ships that would work poorly in the current environment of Galaxy. My idea I had for PvE included specialty ships that would not make sense to have in main, so perhaps this is something worth considering

**Comment by Wingysam** [comment]
What if we just made their gimmicks only apply to the scenario, that way you could still fly them around/heal them in normal servers?

**Comment by Jaxumi92** [comment]
That could work actually, but I'm no dev, so I'm not sure how they'd feel about it

**Comment by PataNautic** [comment]
> What if we just made their gimmicks only apply to the scenario, that way you could still fly them around/heal them in normal servers?

I think this is a fine solution. They don't necessarily need to be viable outside of Stronghold
